po/fr.po

```diff
@@ -1274,7 +1304,7 @@ msgid "Incomplete escape sequence '%s'"
 msgstr ""
 
 msgid "Information request"
-msgstr "Demande d'information"
+msgstr "Demande d’information"
```

Comment:
FWIW in future, it might be beneficial, to put global (but simple) changes such as punctuation changes into a separate commit, to make it easier to review.